### PR TITLE
Fix small todos across codebase

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Fix various small todos in the codebase.

## Justify why they are needed

See inline comments.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1721** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1721" target="_blank">Graphite</a> 
    * **PR #1722** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1722" target="_blank">Graphite</a> 
      * **PR #1725** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1725" target="_blank">Graphite</a> 
        * **PR #1726** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1726" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1726?utm_source=stack-comment).

File: apps/store/src/components/ProductPage/PurchaseForm/ComparisonTableContainer.tsx [hunk 1]
```diff
@@ -55,7 +55,6 @@ const removeDuplicates = <T,>(arr: T[]): T[] => Array.from(new Set(arr))
 const offerHasPeril = (offer: ProductOfferFragment, perilTitle: string) =>
   offer.variant.perils.some((peril) => peril.title === perilTitle)
 
-//@TODO - base on selected offer
```
Comment (robinandeer) [inline apps/store/src/components/ProductPage/PurchaseForm/ComparisonTableContainer.tsx:58]: I think this is a good approach already

File: apps/store/src/components/ProductPage/PurchaseForm/useHandleSubmitAddToCart.ts [hunk 2]
```diff
@@ -18,7 +14,7 @@ type Params = {
 export const useHandleSubmitAddToCart = ({ cartId, onSuccess }: Params) => {
   const [addEntry, { loading }] = useCartEntryAdd({
     // Refetch recommendations
-    refetchQueries: [ShopSessionDocument],
+    refetchQueries: 'active',
```
Comment (robinandeer) [inline apps/store/src/components/ProductPage/PurchaseForm/useHandleSubmitAddToCart.ts:17]: this is more reliable, we have two queries already that fetch product recommendations

File: apps/store/src/pages/_document.tsx [hunk 3]
```diff
@@ -13,7 +13,6 @@ export default class MyDocument extends Document {
     return (
       <Html lang={this.lang()} dir="ltr">
         <Head>
-          {/*TODO: Allow indexing right before going live*/}
```
Comment (robinandeer) [inline apps/store/src/pages/_document.tsx:16]: not sure why this is still here, controlled from the CMS

Comment (alebedev) [reply apps/store/src/pages/_document.tsx:16]: Safe to remove

**Review by alebedev — APPROVED** [review]
Looks good

Not sure why tests are failing, does not seem related

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Job ID: 9U7PtOkCklTQtkcJeLwK.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1726)

**Pull request merged**